### PR TITLE
CLI: Stub console.log/error/info in a way that allows us to print it to stdout

### PR DESCRIPTION
**Pull request by jakubadamw**
# Description

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context.

# Type of change

- [ ] 💔 Breaking
- [X] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

**Review by fbjork — APPROVED** [review]

File: cli/crates/server/src/bridge/server.rs [hunk 1]
```diff
@@ -23,12 +24,12 @@ use std::io::BufReader;
 use std::net::{Ipv4Addr, SocketAddr};
 use std::sync::Arc;
 
-use tokio::sync::broadcast::Sender;
 use tower_http::trace::TraceLayer;
 
 struct HandlerState {
     worker_port: u16,
     pool: SqlitePool,
+    sender: tokio::sync::mpsc::Sender<ServerMessage>,
```
Comment (Finistere) [inline cli/crates/server/src/bridge/server.rs:32]: event_bus?

File: cli/crates/server/src/custom_resolvers.rs [hunk 2]
```diff
@@ -165,13 +144,20 @@ async fn build_resolver(
     )
     .await?;
 
+    let process_env_prelude = format!(
```
Comment (Finistere) [inline cli/crates/server/src/custom_resolvers.rs:147]: I see, that's how you escaped write toml with a proper library! :D

File: cli/crates/server/src/bridge/resolvers.rs [hunk 3]
```diff
@@ -12,14 +16,30 @@ struct ResolverArgs<'a> {
...
+#[derive(serde::Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct ResolverResponse {
+    log_entries: Vec<ResolverMessage>,
+    #[serde(flatten)]
+    rest: serde_json::Value,
+}
+
 pub async fn invoke_resolver(
+    event_bus: &tokio::sync::mpsc::Sender<ServerMessage>,
```
Comment (Finistere) [inline cli/crates/server/src/bridge/resolvers.rs:35]: `bridge_sender` and not event_bus from what I've read later

**Review by Finistere — APPROVED** [review]

File: cli/crates/server/src/custom_resolvers.rs (hunk 2 above)
Comment (jakubadamw) [reply cli/crates/server/src/custom_resolvers.rs:147]: Hehe. No, we just use a different mechanism for env variables, which is required by the DX we want

**Pull request merged**